spicetify.go

```diff
@@ -249,7 +249,7 @@ func main() {
 
 		case "auto":
 			cmd.Auto(version)
-			restartSpotify()
+			restartSpotify(true)
```

Comment:
```suggestion
			cmd.EvalSpotifyRestart(true)
```
would be much simpler fix

Reply:
My reasoning for this is to avoid creating a special case where `noRestart` doesn't work as expected.

Reply:
Well, auto is clearly to backup, apply and restart spotify so I don't believe `noRestart` should be checked there